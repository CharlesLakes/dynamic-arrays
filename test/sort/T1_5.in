10
4 71 30 37 98 3 47 95 100 66