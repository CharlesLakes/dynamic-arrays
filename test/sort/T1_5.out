9 17 30 55 59 77 79 79 85 90